contracts/Liquidator/Liquidator.sol

```diff
@@ -4,11 +4,27 @@ pragma solidity 0.8.13;
 import "@openzeppelin/contracts-upgradeable/token/ERC20/utils/SafeERC20Upgradeable.sol";
 import "@openzeppelin/contracts-upgradeable/security/ReentrancyGuardUpgradeable.sol";
 import "@openzeppelin/contracts-upgradeable/access/Ownable2StepUpgradeable.sol";
+import "@venusprotocol/governance-contracts/contracts/Governance/AccessControlledV8.sol";
+import "./LiquidatorStorage.sol";
 
 interface IComptroller {
+    enum Action {
```

Comment:
Maybe we could set only a constant with the value of `LIQUIDATE`... not a strong opinion here

Reply:
Let's keep it as it is, and in the future we can move the full enum to a single place